README.md

```diff
@@ -1,16 +1,17 @@
 # Percona Monitoring and Management 2.x
+
 [![CI](https://github.com/percona/pmm/actions/workflows/common.yml/badge.svg)](https://github.com/percona/pmm/actions/workflows/common.yml)
 [![CLA assistant](https://cla-assistant.percona.com/readme/badge/percona/pmm)](https://cla-assistant.percona.com/percona/pmm)
 [![Code coverage](https://codecov.io/gh/percona/pmm/branch/main/graph/badge.svg)](https://codecov.io/gh/percona/pmm)
 [![Go Report Card](https://goreportcard.com/badge/github.com/percona/pmm)](https://goreportcard.com/report/github.com/percona/pmm)
 [![Discord](https://img.shields.io/discord/808660945513611334?label=Discord&logo=Discord&style=flat)](https://per.co.na/discord)
 [![Forum](https://img.shields.io/badge/Forum-join-brightgreen)](https://forums.percona.com/)
 
-![PMM](https://www.percona.com/sites/default/files/pmm-logo.png)
+![PMM](https://www.percona.com/wp-content/uploads/2023/01/logo.svg)
 
 ## Percona Monitoring and Management
 
-A 'single pane of glass' to easily view and monitor the performance of your MySQL, MongoDB, PostgreSQL, and MariaDB databases.
+A `single pane of glass` to easily view and monitor the performance of your MySQL, MongoDB, PostgreSQL, and MariaDB databases.
```

Comment:
I'm not sure if we need to change it here.

Reply:
It makes it visible.

Reply:
yeah, but in the markdown view, it's shown like a piece of code and I don't think that it's a good place to use this kind of quota.

Reply:
Ok, I wanted to highlight the expression somehow because now it doesn't look like it stands out even though the intention - I'm pretty sure - was to do make it stand out.

<img width="859" alt="Screenshot 2023-02-15 at 23 44 48" src="https://user-images.githubusercontent.com/81549/219152893-bddea878-93b0-477d-8ec4-f0e91ccd65b7.png">

What if we make it bold? Will it pass?

<img width="1037" alt="Screenshot 2023-02-15 at 23 49 12" src="https://user-images.githubusercontent.com/81549/219154840-a55d5a4c-a12f-411e-ae4d-82204d6d8ee1.png">

Reply:
yeah, making it bold looks better